hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.tsx

```diff
@@ -101,7 +123,7 @@ export default function ActionRuleFormColumns({
       <td>
         <Form.Label>
           Name
-          <span hidden={!showErrors || name}> (required)</span>
+          <span hidden={!showErrors || !!name}> (required)</span>
```

Comment:
`!showErrors || name` returns a string if `showErrors` is true

`!showErrors || !!name` always returns a bool